### PR TITLE
added for cycle

### PR DESCRIPTION
**Pull request by kulverstux**
added for cycle

### CODE REVIEW

File: Learning/Manto/MantoBubbleSort.cs [hunk 1]
```diff
@@ -18,6 +18,14 @@ public static int[] Sort(int[] intArray)
                     }
                 }
             }
+
```
Comment (vidis01) [inline Learning/Manto/MantoBubbleSort.cs:21]: delete not necessary code

File: Learning/Manto/MantoBubbleSort.cs [hunk 2]
```diff
@@ -18,6 +18,14 @@ public static int[] Sort(int[] intArray)
...
+            //blogas kodas
+            for (int a = 1; a < intArray.Length; a++)
+            {
+                Console.WriteLine(intArray[a]);
+            }
+            //blogas kodas
```
Comment (vidis01) [inline Learning/Manto/MantoBubbleSort.cs:27]: delete, not needed and code with error

File: Learning/Program.cs [hunk 3]
```diff
@@ -7,6 +7,7 @@ internal class Program
         static void Main(string[] args)
         {
             Console.OutputEncoding = Encoding.UTF8;
+            //dar vienas
```
Comment (vidis01) [inline Learning/Program.cs:10]: delete

**Review by vidis01 — CHANGES_REQUESTED** [review]

File: Learning/Program.cs (hunk 3 above)
Comment (kulverstux) [reply Learning/Program.cs]: but that comment line looks so good

**Review by vidis01 — APPROVED** [review]

**Pull request merged**